test/testAdaptors/AuraERC4626Adaptor.t.sol

```diff
@@ -0,0 +1,292 @@
+        );
+        assertApproxEqAbs(
+            auraRETHWETHBPTVault.balanceOf(address(cellar)),
+            0,
+            2,
+            "Cellar should have redeemed all vault aura share tokens."
+        );
+    }
+
+    // same as redeem test but with a long time within the vault. Shows that rewards are not the vault asset (BPT) but other tokens.
+    function testFastForward(uint256 assets) external {
+        assets = bound(assets, 0.1e18, 1_000_000_000e18);
+        deal(address(rETH_wETH_BPT), address(this), assets);
+        cellar.deposit(assets, address(this)); // bpts sent to aura pool as seen in deposit() tests
+
+        skip(100 days);
+        mockWethUsd.setMockUpdatedAt(block.timestamp);
+        mockRethUsd.setMockUpdatedAt(block.timestamp);
+
+        uint256 maxRedeem = cellar.maxRedeem(address(this));
+
+        uint256 redeemedAssets = cellar.redeem(maxRedeem, address(this), address(this)); // state mutative call to redeem assets to address(this)
+
+        assertApproxEqAbs(
+            assets,
+            redeemedAssets,
+            2,
+            "BPT balance should not change, yield is earned via other ERC20s (AURA, BAL, etc.)."
+        );
+        assertApproxEqAbs(
+            rETH_wETH_BPT.balanceOf(address(this)),
+            assets,
+            1,
+            "All BPTs should have been returned to user."
+        );
+        assertApproxEqAbs(
+            auraRETHWETHBPTVault.balanceOf(address(cellar)),
+            0,
+            2,
+            "Cellar should have redeemed all vault aura share tokens."
+        );
+    }
+
+    function testInterestAccrual(uint256 assets) external {
+        assets = bound(assets, 0.1e18, 1_000_000_000e18);
+        deal(address(rETH_wETH_BPT), address(this), assets);
+        cellar.deposit(assets, address(this)); // bpts sent to aura pool as seen in deposit() tests
+
+        vm.warp(block.timestamp + 100 days); // fast forward to accrue rewards
+        mockWethUsd.setMockUpdatedAt(block.timestamp);
+        mockRethUsd.setMockUpdatedAt(block.timestamp);
+
+        uint256 oldBALRewards = BAL.balanceOf(address(cellar));
+        uint256 oldAURARewards = AURA.balanceOf(address(cellar));
+        uint256 oldBPTBalance = rETH_wETH_BPT.balanceOf(address(cellar));
+
+        Cellar.AdaptorCall[] memory data = new Cellar.AdaptorCall[](1);
+        // Withdraw FRAX from FraxLend.
+        bool claimExtras = true;
+        {
+            bytes[] memory adaptorCalls = new bytes[](1);
+            adaptorCalls[0] = _createBytesDataGetRewardsFromAuraPoolERC4626(address(aura_rETH_wETH_BPT), claimExtras);
+            data[0] = Cellar.AdaptorCall({ adaptor: address(auraERC4626Adaptor), callData: adaptorCalls });
+        }
+
+        // Perform callOnAdaptor.
+        cellar.callOnAdaptor(data); // TODO: EIN, check the logs to see what reward tokens are claimed with bool set to false. Then compare to what it is as true.
```

Comment:
Did you do this?

Reply:
Yes, I double checked by running console logs on the balances. These console logs are the same that were mentioned in your other comment before within the actual adaptor design.